#goal definition
uint32 task_number
int32 error_step
int32 error_object
---
#result definition
uint8 status
bool is_complete
---
#feedback
float32 x
float32 y
float32 z
uint8 status
string text
